api/location/schema.py

```diff
@@ -82,6 +82,23 @@ def mutate(self, info, location_id, **kwargs):
         return UpdateLocation(location=location_object)
 
 
+class DeleteLocation(graphene.Mutation):
+    class Arguments:
+        location_id = graphene.Int(required=True)
+
+    location = graphene.Field(Location)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, location_id, **kwargs):
+        query_location = Location.get_query(info)
+        exact_location = query_location.filter(
```

Comment:
I would suggest you go with simple variable names. `query_location` could just be `query`, `exact_location` could just be `location`.

Reply:
@joshuaocero , I just made the changes you requested